File: app/src/main/java/com/njamb/geodrink/Classes/FriendViewHolder.java
package com.njamb.geodrink.Classes;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.njamb.geodrink.R;


public class FriendViewHolder extends RecyclerView.ViewHolder {
    public TextView tvFullName;
    public TextView tvUsername;
    public TextView tvEmail;
    public ImageView ivProfileImg;

    public FriendViewHolder(View itemView) {
        super(itemView);

        tvFullName = (TextView) itemView.findViewById(R.id.friend_full_name);
        tvUsername = (TextView) itemView.findViewById(R.id.friend_username);
        tvEmail = (TextView) itemView.findViewById(R.id.friend_email);
        ivProfileImg = (ImageView) itemView.findViewById(R.id.friend_profile_pic);
    }
}

File: app/src/main/java/com/njamb/geodrink/view_holders/FriendViewHolder.kt
package com.njamb.geodrink.view_holders

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import android.widget.TextView

import com.njamb.geodrink.R


class FriendViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var tvFullName: TextView
    var tvUsername: TextView
    var tvEmail: TextView
    var ivProfileImg: ImageView

    init {

        tvFullName = itemView.findViewById(R.id.friend_full_name) as TextView
        tvUsername = itemView.findViewById(R.id.friend_username) as TextView
        tvEmail = itemView.findViewById(R.id.friend_email) as TextView
        ivProfileImg = itemView.findViewById(R.id.friend_profile_pic) as ImageView
    }
}
